### PR TITLE
Rebase of firmware_flasher.js

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixed code styling, etc in `firmware_flasher.js`

### CODE REVIEW

**Comment by haslinghuis** [comment]
Please wait for approvals as this will probably needs rebasing after mentioned PR's are merged.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Rebased after merging 2501

**Review by chmelevskij — DISMISSED** [review]

**Review by chmelevskij — APPROVED** [review]

File: src/js/tabs/firmware_flasher.js
Comment (Asizon) [inline src/js/tabs/firmware_flasher.js:564]: @haslinghuis i think "is" should be removed in the comment?

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - Github identifies PR as mergeable -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
